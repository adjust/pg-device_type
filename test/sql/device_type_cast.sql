BEGIN;
CREATE EXTENSION device_type;
SELECT 'server'::device_type::int4;
SELECT 'server'::device_type::int2;
SELECT 160::int4::device_type;
SELECT 160::int2::device_type;
ROLLBACK;
