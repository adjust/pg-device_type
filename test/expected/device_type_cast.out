BEGIN;
CREATE EXTENSION device_type;
SELECT 'server'::device_type::int4;
 int4 
------
  160
(1 row)

SELECT 'server'::device_type::int2;
 int2 
------
  160
(1 row)

SELECT 160::int4::device_type;
 device_type 
-------------
 server
(1 row)

SELECT 160::int2::device_type;
 device_type 
-------------
 server
(1 row)

ROLLBACK;
